src/styles/_mixin.scss

```diff
@@ -0,0 +1,7 @@
+@mixin responsive ($map, $width) {
+  @each $size, $columns in $map {
+    @media (max-width: $size) {
+      grid-template-columns: repeat($columns, $width);
+    }
+  }
+}
```

Comment:
it's amazing!!! now I also want to do in such way😁🥺

Reply:
Thank you!I appreciate that 🥰